qlib/data/data.py

```diff
@@ -588,11 +573,19 @@ def _load_instruments(self, market):
         if not os.path.exists(fname):
             raise ValueError("instruments not exists for market " + market)
         _instruments = dict()
-        df = pd.read_csv(fname, sep="\t", names=["inst", "start_datetime", "end_datetime", "save_inst"])
-        df["start_datetime"] = pd.to_datetime(df["start_datetime"])
-        df["end_datetime"] = pd.to_datetime(df["end_datetime"])
-        for row in df.itertuples(index=False):
-            _instruments.setdefault(row[0], []).append((row[1], row[2]))
+        with open(fname) as f:
+            for line in f:
+                inst_time = line.strip().split()
+                inst = inst_time[0]
+                if len(inst_time) == 3:
+                    # `day`
+                    begin = inst_time[1]
+                    end = inst_time[2]
+                elif len(inst_time) == 5:
```

Comment:
This is very strange to use different logic for processing  different frequency data.
Why not using  code like  `pd.read_csv(fname, sep="\t", names=["inst", "start_datetime", "end_datetime"])`